app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Nice job on configuring devise authentication parameters correctly 👌 however, I'm unable to successfully register a new user due to validation errors, as can been seen in the screenshot below. We can improve upon this aspect by including the _name_ input field in the sign-up form adding `allow_blank: true` to the validations of _posts_counter_.

**Sign up error:**
![image](https://github.com/emarkees/Gist_Blog/assets/12745474/d8726ff7-0d82-436c-bc3f-b1cfd9dd144b)